src/runmode-unix-socket.c

```diff
@@ -744,6 +744,100 @@ TmEcode UnixSocketDatasetRemove(json_t *cmd, json_t* answer, void *data)
     }
 }
 
+TmEcode UnixSocketDatasetDump(json_t *cmd, json_t *answer, void *data)
+{
+    SCEnter();
+    SCLogDebug("Going to dump datasets");
+    DatasetsSave();
```

Comment:
Should we check if `save` command is used in the loaded datasets? I used it with a `load` command dataset and got the output
```
Success:
"datasets dump done"
```
but it seems to move to next entry if `save` is not found so it did not seem to do anything.

Reply:
I don't think it is an issue as it is just a noop in this case. For example, you can use the command to dump periodically the datasets (via cron) and in this case you don't really control which are the datasets used and don't really want a warning.